guide/C/ch_loans.xml

```diff
@@ -27,82 +27,93 @@
     </para>
 
     <sect2 id="loans_conceptsterms2">
-      <title>Terminology</title>
```

Comment:
Can you teach your editor to use 2 spaces for indentation instead of tabs? Or run xmlformat - I hope I have the time to fix it soon.